modules/node/src/transfer/transfer.service.ts

```diff
@@ -369,14 +369,14 @@ export class TransferService {
     return res.appInstanceId;
   }
 
-  // TODO: why don't these functions throw errors, this breaks the patterns we normally use
   async reclaimLinkedTransferCollateralByAppInstanceIdIfExists(
     appInstanceId: string,
   ): Promise<void> {
     const transfer = await this.linkedTransferRepository.findByReceiverAppInstanceId(appInstanceId);
     if (!transfer || transfer.status !== LinkedTransferStatus.REDEEMED) {
-      this.log.debug(`Did not find transfer`);
-      return;
+      throw new Error(
```

Comment:
But then this will throw errors every time an app is uninstalled right? Since we call this in the listener.

Reply:
handled these errors there